py3status/modules/xrandr_rotate.py

```diff
@@ -89,7 +77,7 @@ def _apply(self):
         outputs = [self.screen] if self.screen else self._get_all_outputs()
         for output in outputs:
             cmd = 'exec xrandr --output ' + output + ' --rotate ' + rotation
-            Popen(['i3-msg', cmd], stdout=PIPE)
+            self.py3.command_run(['i3-msg', cmd])
```

Comment:
I wonder if we're keeping `exec` + `i3-msg` here for a good reason or maybe it's just something we should remove... Run command directly instead?